vertical-pod-autoscaler/pkg/updater/logic/updater.go

```diff
@@ -155,6 +165,19 @@ func (u *updater) RunOnce() {
 	timer.ObserveTotal()
 }
 
+func getRateLimiter(evictionRateLimit float64, evictionRateLimitBurst int) *rate.Limiter {
+	var evictionRateLimiter *rate.Limiter
+	if evictionRateLimit == -1 || evictionRateLimit == 0 {
```

Comment:
What happens if evictionRateLimit is < 0?

Reply:
I changed the condition 👍